docs/explanation/dss-artefacts.rst

```diff
@@ -0,0 +1,10 @@
+DSS Artefacts
+=============
+
+Notebook files
+--------------
+DSS Notebook .ipynb files are stored in the DSS shared volume.
+
+Models
```

Comment:
So there are also other MLflow artifacts which users can and will store e.g. experiments and runs. Should we also mention those ? Please sync with @kimwnasptd .

Reply:
And also, let's have a small note in each section how a user could move those artifacts from the DSS to their machine. We could instruct them to use the Jupyter and MLflow UIs, and for Notebooks to also access the shared volume (link to docs)

Reply:
>  move those artifacts from the DSS to their machine

I don't think this is accurate, the files are already on their machine
we can tell them how to access the mounted volume by linking to the access data guide, wdyt?

Reply:
What if someone would need to move a versioned trained model from MLflow "outside of DSS" so that they can then upload it to CKF or anything.

We'd need a way to tell them how to get those files. It doesn't have to be something super complicated, and even telling them they can download them from UI is fine. But we need a way to tell them what are the artifacts they would be interested in and how to "get them out of DSS"

Reply:
I made some changes, let me know if this is sufficient

Reply:
I'm judging this is good enough for a first pass.  If @kimwnasptd or @misohu want more, we can add it as a separate PR